divide_and_conquer/convex_hull.py

```diff
@@ -88,7 +88,7 @@ def __le__(self, other):
         return False
 
     def __repr__(self):
-        return "({}, {})".format(self.x, self.y)
+        return f"({self.x}, {self.y})"
```

Comment:
__return f"Point({self.x}, {self.y})"__

__repr__() should _return a string that would yield an object with the same value when passed to eval()_
https://docs.python.org/3/library/functions.html#repr